spec/std/http/spec_helper.cr

```diff
@@ -11,12 +11,12 @@ private def wait_for(&)
             {% else %}
               5.seconds
             {% end %}
-  now = Time.monotonic
+  now = Time.instant
 
   until yield
     Fiber.yield
 
-    if (Time.monotonic - now) > timeout
+    if now.elapsed > timeout
```

Comment:
*Thought*: the pattern keeps repeating; maybe we'd like a `Time::Instant#elapsed?(Time::Span)` method?

```crystal
timer = Time.instant

if timer.elapsed?(timeout)
  # ...
end
```